python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -48,8 +48,25 @@ class CliLog:
 
     defaultLsstLogLevel = lsstLog.FATAL if lsstLog is not None else None
 
-    longLogFmt = "%-5p %d{yyyy-MM-ddTHH:mm:ss.SSSZ} %c (%X{LABEL})(%F:%L)- %m%n"
-    normalLogFmt = "%c %p: %m%n"
+    lsstLog_longLogFmt = "%-5p %d{yyyy-MM-ddTHH:mm:ss.SSSZ} %c (%X{LABEL})(%F:%L)- %m%n"
+    """The log format used when the lsst.log package is importable and the log
+    is initialized with longlog=True."""
+
+    lsstLog_normalLogFmt = "%c %p: %m%n"
+    """The log format used when the lsst.log package is importable and the log
+    is initialized with longlog=False."""
+
+    pylog_longLogFmt = "%(levelname)s %(asctime)s %(name)s %(filename)s:%(lineno)s - %(message)s"
+    """The log format used when the lsst.log package is not importable and the
```

Comment:
This is quite  hard to understand without the hidden context below. Naively you would think that python logging should be configured regardless of lsst.log. Somehow we need to make it clearer that we actually bypass python logging if lsst.log is available -- not simply that we decide to use different log formats.

Reply:
I will add to the class docstring about this.